app/assets/javascripts/discourse/app/lib/download-calendar.js

```diff
@@ -37,15 +37,18 @@ export function downloadIcs(title, dates) {
   setTimeout(() => window.URL.revokeObjectURL(file), REMOVE_FILE_AFTER); //remove file to avoid memory leaks
 }
 
-export function downloadGoogle(title, dates) {
+export function downloadGoogle(title, dates, recurrenceRule) {
   dates.forEach((date) => {
     const encodedTitle = encodeURIComponent(title);
-    const link = getURL(`
-      https://www.google.com/calendar/event?action=TEMPLATE&text=${encodedTitle}&dates=${_formatDateForGoogleApi(
+    let link = `https://www.google.com/calendar/event?action=TEMPLATE&text=${encodedTitle}&dates=${_formatDateForGoogleApi(
```

Comment:
I looked up and found there is a much nicer way to construct these URLs especially if you have conditionals:

https://www.valentinog.com/blog/url/

```javascript
const link = new URL("https://www.google.com/calendar/event")
link.searchParams.append("action", "TEMPLATE")
link.searchParams.append("text", "blah")
getUrl(link.href)

=> 'https://www.google.com/calendar/event?action=TEMPLATE&text=blah'
```